tests/AssertExceptionClassTest.php

```diff
@@ -12,63 +12,63 @@ public function testDefault()
 		$test = function() {
 			throw new Exception();
 		};
-		AssertException::assertException($test, null);
+		$this->assertInstanceOf(Exception::class, AssertException::assertException($test, null));
```

Comment:
IMO, the return type should not be tested in every test case:
If `assertException` does not return the expected type, then every test will fail – not very useful.

I'd suggest to test that in a single place, e.g. `testReturnType()`.